package.json

```diff
@@ -19,6 +19,8 @@
     "@vercel/analytics": "^0.1.6",
     "dexie": "^3.2.2",
     "dexie-react-hooks": "^1.1.1",
+    "html2canvas": "^1.4.1",
+    "jspdf": "^2.5.1",
```

Comment:
I think you're not using the jspdf, better to remove it
cause it will only make the bundle size alot bigger, since it's size is 14.7MB which is quite big